src/now.js

```diff
@@ -451,7 +461,7 @@ const main = async (argv_) => {
     // Only download team data if not cached
     if (!cachedTeam && !cachedUser) {
       if (isTTY) {
-        console.log(info('Caching team information'))
```

Comment:
Actually, I don't see any value in this message and the other one. Let's get rid of them